kafka-rest/src/main/java/io/confluent/kafkarest/controllers/TopicConfigManager.java

```diff
@@ -15,39 +15,37 @@
 
 package io.confluent.kafkarest.controllers;
 
-import io.confluent.kafkarest.entities.TopicConfiguration;
+import io.confluent.kafkarest.entities.TopicConfig;
```

Comment:
I'd call this `TopicConfigProperty` (configuration seems a bit long winded here).